packages/docz-tools/src/gatsby-theme-docz/components/Layout/index.tsx

```diff
@@ -11,7 +11,7 @@ import "@jobber/design/foundation.css";
 
 export function Layout({ children }: PropsWithChildren<{}>) {
   const {
-    themeConfig: { sideBarWidth, containerWidth },
+    themeConfig: { sideBarWidth, containerWidth, hideActions },
```

Comment:
maybe `hasActions` is a better name here?

Reply:
Edited so that `hideActions` is now `hasActions`. I have edited the `doczrc.js` file in Atlantis so that it is defaulted to `true`.